plugins/sanity-plugin/studio/package.json

```diff
@@ -28,6 +28,7 @@
     "prop-types": "^15.7",
     "react": "^17.0",
     "react-dom": "^17.0",
+    "sanity-plugin-gatsby-cloud-preview": "^1.0.3",
```

Comment:
@kathmbeck @DanielSLew I'm missing the context here. Is this required when any site uses `gatsby-source-sanity`? I'm not seeing anything about this package mentioned in the README

Reply:
It's needed for content sync to be available out of the box. It's more so that we indicate that's the case for the starter here: https://www.npmjs.com/package/sanity-plugin-gatsby-cloud-preview